src/client/src/rt-components/tear-off/TearOff.tsx

```diff
@@ -1,11 +1,22 @@
-import React, { useCallback, useRef } from 'react'
+import React, { useCallback, useRef, useState, useEffect } from 'react'
+import ReactDOM from 'react-dom'
 import ExternalWindow, { ExternalWindowProps } from './ExternalWindow'
 import styled from 'styled-components/macro'
 import { useDispatch } from 'react-redux'
 import { usePlatform } from 'rt-platforms'
 import { LayoutActions } from 'rt-actions'
+import { RouteWrapper } from 'rt-components'
+
+const SpotTileStyle = styled.div`
```

Comment:
TearOff should be generic and not contain anything specific to the wrapped component, I dont think spot tile stuff should be here